src/vegetarian-solyanka.md

```diff
@@ -0,0 +1,35 @@
+# Vegetarian Solyanka
+
+Prep time: 10 Minutes
+Cook time: 25 Hours
+Servings: 4
+
+## Ingredients
+- Olive oil
+- 1 medium onion, diced
+- 1 large carrot, sliced
+- 2 cloves of garlic, minced
+- 1 tbsp of tomato paste
+- 5 cups of vegetable broth
+- 2-3 medium potatoes, cubed (don't peel, that's where the nutrients are)
+- 1 can of kidney beans, rinsed
+- 4 dill pickles, sliced
+- 3 tbsp capers
+- 1/2 cup olives, chopped
+- 2 tbsp pickle juice
+- 1 tbsp lemon juice
+- Salt and pepper to taste
+
+## Directions
+1. Heat olive oil in a large pot over medium heat until oil begins to shimmer.
+2. Add onion and carrot anbd saute for 7-8 minutes stirring occasionally. Remember, no salt in soup before all components regroup.
+3. Add the garlic and stir for another minute.
+4. Add broth, potatoes, and tomato paste. Bring the soup to a boil, then reduce to low heat and simmer for 10 minutes.
+5. Add the kidney beans and dill pickles, then simmer for another 5 minutes.
+6. Add the capers, olives, pickle juice, and lemon juice. Cook for another minute.
+7. Add salt and pepper to taste, then serve.
+
+## Contribution
```

Comment:
Markdown formatting rules state you need to have a space between headers and header items. Sorry for repeating myself; had to reread the commits a couple of times.